### PR TITLE
Fix stt header check for stormpath JWTs

### PR DESCRIPTION
**Pull request by jarias**

### CODE REVIEW

**Comment by dogeared** [comment]
@jarias - please update `DefaultAccessTokenTest.testInvalidAccessToken` as well. Then, this can be merged once CI passes.

**Comment by jarias** [comment]
PR updated

File: impl/src/test/groovy/com/stormpath/sdk/impl/oauth/DefaultAccessTokenTest.groovy [hunk 1]
```diff
@@ -148,31 +153,44 @@ class DefaultAccessTokenTest {
...
-        String jwt = Jwts.builder()
-            .setSubject(href)
-            .claim("stt", "refresh")
-            .signWith(SignatureAlgorithm.HS256, secret.getBytes("UTF-8"))
-            .compact();
-
-        def properties = [
-            href: href,
-            jwt: jwt
+        def invalidJwts = [
```
Comment (dogeared) [inline impl/src/test/groovy/com/stormpath/sdk/impl/oauth/DefaultAccessTokenTest.groovy]: lovely!

**Comment by dogeared** [comment]
LGTM - please merge after CI passes.

**Pull request merged**